### ISSUE TITLE
Particle systems set to Hair are handled by creating objects for each instance

### ISSUE THREAD

**Issue by briligg** [xavier150/Blender-For-UnrealEngine-Addons#12]
I just wanted to see what would happen if I tried to export an object with the particle system intact. It hadn't shown any potential error when i did the check. I had to do a hard shut-down after it spent a while trying to convert the 10 000 rocks scattered around the object's surface into separate child objects. At least i think that's what it was doing, it never finished. 

It was perhaps a foolish experiment, but i thought i should mention it.

**Comment by xavier150** [comment]
Haha, yes that is not part of the potential error because the particle system can also be used to add 3D elements to an object

**Comment by VexisMorlock** [comment]
Is there a way to export objects without creating a new object for each particle.  ie a donuts with sprinkles as hair.  The whole reason I used the particle system to begin with was so I didn't have to place them all by hand.

**Comment by xavier150** [comment]
Of course, you just need to export an object as you normally do with the particle system activated
http://xavierloux.com/sharex/ApplicationFrameHost_2019-05-18_16-37-11.jpg

**Comment by VexisMorlock** [comment]
I think I am having an issue while importing, or I am showing my novice.  If I open the export in 3d paint its intact but when I import it into ure4 it separates everything.
ie https://i1168.photobucket.com/albums/r494/bettymorlock/ure4%20importing%20issue_zps6rbxmzvj.png

**Comment by xavier150** [comment]
Oh yes, for Static mesh assets you need tick CombineMeshs in FBX Import Options :)
https://github.com/xavier150/Blender-For-UnrealEngine-Addons/blob/master/Tuto/How%20import%20assets%20from%20Blender%20to%20Unreal.md

**Comment by VexisMorlock** [comment]
Awesome thanks that worked.  I don't know how I missed that; appreciate the help.
